src/field_advance/standard/pipeline/compute_div_b_err_pipeline.h

```diff
@@ -13,7 +13,7 @@ typedef struct pipeline_args
   const grid_t *              g;
 } pipeline_args_t;
 
-#define f(x,y,z) f[ VOXEL(x,y,z, nx,ny,nz) ]
+#define f(x,y,z) f[ VOXEL( x, y, z, nx, ny, nz ) ]
```

Comment:
Maybe it's time to define a style guide. I think this should be a third option:

` VOXEL(x, y, z, nx, ny, nz)`

Reply:
After consulting clang format, which I want to move us to eventually, I'm fine with how you have it!